src/dbt_osmosis/core/osmosis.py

```diff
@@ -1049,6 +1052,31 @@ def update_columns_data_type(
                             changes_committed += 1
         return changes_committed
 
+    def update_columns_description(
```

Comment:
IMO: The logic of the code is almost the same as `update_columns_data_type`. Since having the same kind of code all over the place is a code smell, I want to abstract it and test it.

Reply:
@syou6162 
Thank you for the feedback!
I got it, I will work on fixing it.